test/unit/test_autosubmit.py

```diff
@@ -0,0 +1,41 @@
+import os
+from textwrap import dedent
+from unittest.mock import patch
+from pathlib import Path
+
+from autosubmitconfigparser.config.basicconfig import BasicConfig
+
+from autosubmit.autosubmit import Autosubmit
+
+def test_copy_as_config(tmp_path):
```

Comment:
You probably want to use one of our fixtures that configure Autosubmit for you... Look at `conftest.py`.